test/e2e_test.go

```diff
@@ -0,0 +1,230 @@
+package test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/drone/envsubst"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestDecco(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Decco E2E Suite")
+}
+
+const (
+	testStagingDir            = "./test_staging"
+	testStagingManifestsDir   = testStagingDir + "/manifests"
+	testDataDir               = "./test_data"
```

Comment:
You can precreate a `./testdata` dir and it won't be included in the builds. 
https://medium.com/rungo/unit-testing-made-easy-in-go-25077669318 (look for Test Data section). Used it heavily in nodelet to hold files that I will use during unit tests.
You can then avoid all makedirs...

Reply:
Thanks @pacharya-pf9 - I will take a look 🙂

Reply:
I changed test_data to testdata and moved all permanent and temporary test data into that directory